src/index.html

```diff
@@ -13,18 +13,9 @@ <h1>Today's To Do</h1>
         <input type="text" name="" class="addTask" id="inputTask" placeholder="Add a task...">
         <button id="button">Add</button>
       </div>
-      <!-- <ul id="todo-list"></ul> -->
-      <div class="container">
-        <div class="content">
-          <div class="toDoList">
-            <input type="checkbox" name="" id="">
-            <input type="text" name="" id="" value="text">
-          </div>
-          <i class="fa-regular fa-ellipsis-vertical"></i>
-        </div>
-      </div>
-      <div class="button-container">
-        <button id="clearButton" class="clear">Clear as Complete</button>
+      <ul id="todo-list"></ul>
+      <div class="button-container footer">
```

Comment:
- [ ] @SAZZAD1Q2, am not sure your application is adding or executing any action, please investigate on this error below and resubmit your work 

<img width="901" alt="1" src="https://github.com/SAZZAD1Q2/To-Do-List/assets/77843050/ec85c975-4ab3-47ba-938c-bb88c3e242fe">